### ISSUE TITLE
[Suggestion} Support for jazztronauts gamemode

### ISSUE THREAD

**Issue by BlazeTheFurry** [LeadKiller/leadbot#33]
Gamemode: https://steamcommunity.com/sharedfiles/filedetails/?id=1452613192&searchtext=jazztronauts

**Comment by KoolenDasheppi** [comment]
I agree, Jazztronauts is amazing and although it's playable in singleplayer sorta, it's much more fun with extra players.
